### PR TITLE
base Argon components p2

### PR DESCRIPTION
**Pull request by quanghung97**
https://edu-redmine.sun-asterisk.vn/issues/8608

### CODE REVIEW

**Comment by quanghung97** [comment]
ready

File: nuxt/components/argon/Modal.vue [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+      role="dialog"
+      class="modal fade"
+      @click.self="closeModal"
+    >
+      <div
+        :class="[{'modal-notice': type === 'notice'}, modalClasses]"
+        class="modal-dialog modal-dialog-centered"
+      >
+        <div
+          :class="[gradient ? `bg-gradient-${gradient}` : '',modalContentClasses]"
```
Comment (BunnyPi04) [inline nuxt/components/argon/Modal.vue]: convention

File: nuxt/layouts/other.vue [hunk 2]
```diff
@@ -0,0 +1,383 @@
...
+              href="https://www.facebook.com/hungcan997"
+              class="btn btn-neutral btn-icon-only btn-facebook btn-round btn-lg"
+              data-toggle="tooltip"
+              data-original-title="Like us"
+            >
+              <i class="fa fa-facebook-square"></i>
+            </a>
+            <a
+              target="_blank"
+              href="https://dribbble.com"
```
Comment (BunnyPi04) [inline nuxt/layouts/other.vue]: link sang dribble làm gì

File: nuxt/locales/vi.json [hunk 3]
```diff
@@ -0,0 +1,172 @@
+{
+  "links": {
```
Comment (BunnyPi04) [inline nuxt/locales/vi.json]: indent = 4 spaces

**Pull request merged**